rpcq/_spec.py

```diff
@@ -128,7 +128,7 @@ def get_handler(self, request):
                         return rpc_error(request.id, "{}\n{}".format(str(e), _traceback),
                                          warnings=warnings)
                     else:
-                        return rpc_error(request.id, str(e), warnings=warnings)
+                        return rpc_error(request.id, repr(e), warnings=warnings)
```

Comment:
Maybe the `provide_tracebacks` branch above should also print the `repr`. The traceback provides enough context to disambiguate in any case, but it wouldn't hurt and it's nice that the only difference between the two cases is the traceback getting included or not.

Reply:
> Maybe the `provide_tracebacks` branch above should also print the `repr`. The traceback provides enough context to disambiguate in any case, but it wouldn't hurt and it's nice that the only difference between the two cases is the traceback getting included or not.

I think I need some coffee 😵

Reply:
Yeah, why not.